package.json

```diff
@@ -17,6 +17,7 @@
     "@apollo/react-ssr": "^3.1.3",
     "@formatjs/intl-relativetimeformat": "4.5.10",
     "@material-ui/core": "4.9.7",
+    "@n8tb1t/use-scroll-position": "^1.0.43",
```

Comment:
What is that package, could it be avoided ?

Reply:
It is a small react-hook for scrolling down effects, it is processing the logic related to docking the banner over the footer.
I can rewrite that using `useEffect` but we'll be able to deprecate that with the banner when we don't use it anymore.

Reply:
You can try using css `position: sticky`, it's what we use for the new collective page and in other places. The pure-css approach has better performances and a good browser support nowadays.

Reply:
A few tips if you go with that (I struggled with them):
- Make sure the parent has `position: relative`
- Make sure to set a `top` (or `bottom` based on the behavior you want)

Reply:
Nice, I'll try that.